lamindb/core/writelog/_db_metadata_wrapper.py

```diff
@@ -99,29 +99,11 @@ def _get_columns_by_name(
 
         return column_list
 
-    def _get_columns_by_name(
-        self, table: str, column_names: list[str], cursor: CursorWrapper
-    ) -> list[Column]:
-        columns = self.get_columns(table=table, cursor=cursor)
-
-        column_list: list[Column] = []
-
-        for column_name in column_names:
-            column = next((c for c in columns if c.name == column_name), None)
-
-            if column is None:
-                raise ValueError(
-                    f"Table '{table}' doesn't have a column named '{column_name}'"
-                )
-
-            column_list.append(column)
-
-        return column_list
-
     def get_uid_columns(self, table: str, cursor: CursorWrapper) -> UIDColumns:
         """Get the UID columns for a given table."""
         if table == "lamindb_featurevalue":
-            # TODO: update this to feature + hash instead of value + created_at
+            # TODO: update this to feature uid + hash instead of value + created_at
+            # feature_id is a foreign key on the featurevalue table, so this should be easy
```

Comment:
This is the one thing that I can't figure out but I hope is simple for you given you solved the much harder case of the ManyToMany a few lines below, @alexras!

Can you please update the `TableUID` object so that it reflects `feature.uid` and `featurevalue.hash`. `lamindb_featurevalue.feature` is an FK to `lamindb_feature`.